tests/e2e/utils.js

```diff
@@ -0,0 +1,23 @@
+import { browser, $ } from '@wdio/globals';
+
+let extensionId = '';
+export async function getExtensionId() {
+  if (!extensionId) {
+    await browser.url('chrome://extensions');
+
+    const extension = await $('extensions-item:first-child');
+    extensionId = await extension.getAttribute('id');
+  }
+
+  return extensionId;
+}
+
+export async function navigateToPage(page, file = 'index.html') {
+  await browser.url(
+    `chrome-extension://${await getExtensionId()}/pages/${page}/${file}`,
+  );
+}
+
+export function getQAElement(qa) {
+  return $(`[data-qa="${qa}"]`);
```

Comment:
I think the common pattern is to call this attribute `data-testid`

Reply:
to me QA (Quality Assurance, is quite bit more than tests) but I also don't have a strong opinion. `qa` is short, that's definitely a benefit. `data-testid` is a convention I saw in other projects, I think it comes from https://testing-library.com/docs/queries/bytestid/

how about we simply use regular `id`s ?

Reply:
That was my first idea. However, sometimes we need that for another purpose (usual CSS/JS stuff), and the `id` can be only one.